ftw/participation/tests/test_invite.py

```diff
@@ -111,6 +112,28 @@ def test_inviting_a_user_by_userid(self, browser):
         self.assertIn('Hi th=C3=B6re', mail)
 
     @browsing
+    def test_inviting_a_user_by_userid_with_a_different_login_name_is_possible(self, browser):
+        user = create(Builder('user')
+                      .named('Hugo', 'Boss')
+                      .with_userid('hugo.boss')
+                      .with_email('hugo.boss@example.ch'))
+        api.portal.get_tool('acl_users').source_users.updateUser('hugo.boss', 'hugo.boss@example.ch')
```

Comment:
👍  I added a comment who is explaining how we do achieve this issue in a real plone-site.